<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnJuegos">
       <property name="text">
        <string>Juegos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCategorias">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Categorias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnJuegosFavoritos">
       <property name="text">
        <string>Mis Juegos Favoritos</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>1001</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QComboBox" name="listClassRef">
    <property name="geometry">
     <rect>
      <x>891</x>
      <y>40</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddClass">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AÃ±adir</string>
    </property>
   </widget>
   <widget class="QLabel" name="textNameFrame">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayouSorts"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutFilter"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
